packages/mui-system/src/cssVars/createCssVarsProvider.js

```diff
@@ -156,7 +157,19 @@ export default function createCssVarsProvider(options) {
         return defaultColorScheme.light;
       })();
       if (key === resolvedDefaultColorScheme) {
+        const excludedVariables = {};
```

Comment:
If this turns out to be expensive in the future, we could go with the other way around and have an object of the keys that needs to be deleted, so that we can go over that array, instead of the whole `css`.

Reply:
Good point. I think it is worth updating now.